content/04.study.md

```diff
@@ -72,15 +72,13 @@ required data to be split randomly into two distinct models and trained
 separately. It is unclear how much performance would have increased if not for
 computational restrictions.
 
-Epigenetic data, combined with deep learning, may have sufficient explanatory
-power to infer gene expression. For instance, the DeepChrome CNN [@tag:Singh2016_deepchrome]
-improved prediction accuracy of high or low gene expression from histone modifications over existing
-methods. Deep learning can also integrate different data types. For example,
-Liang et al. combined RBMs to integrate gene expression, DNA methylation, and
-miRNA data to define ovarian cancer subtypes [@tag:Liang2015_exprs_cancer].
-While these approaches are promising, many convert gene expression measurements
-to categorical or binary variables, thus ablating many complex gene expression
-signatures present in intermediate and relative numbers.
+Epigenetic data, combined with deep learning, may have sufficient explanatory power to infer gene expression.
+For instance, a convolutional neural network applied to histone modifications, termed DeepChrome, improved prediction accuracy of high or low gene expression over existing non-neural network
```

Comment:
This edit is reverting a change from #618.  We should keep the existing DeepChrome sentence and add the AttentiveChrome reference (changing `is` to `was`).

Reply:
Agree